catalog/url_listener_test.go

```diff
@@ -47,36 +50,57 @@ func Test_prepareCookieJar(t *testing.T) {
 }
 
 func Test_Listen(t *testing.T) {
-	Convey("Listen()", t, func() {
+	Convey("Listen()", t, func(c C) {
```

Comment:
I did not know this was a thing. Interesting.

Reply:
It's handy for testing inside goroutines, although I still haven't gotten around to fixing this bug: https://github.com/smartystreets/goconvey/issues/477